.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Which Auth library and version are you using ?**
+Note that to get help, you need to run the latest preview or non-preview version
+<!-- E.g. ADAL 3.19.8, MSAL 1.1.4-preview -->
+
```

Comment:
Which application type? and scenario
- Desktop 
  - interactive
  - WIA
  - U/P
  - Device code flow (browserless)
- Mobile 
- Web App 
  - authorization code
  - OBO
- Web API 
  - OBO